src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+  constructor(name) {
+    this.name = name;
```

Comment:
```suggestion
  constructor(name, health = DEFAULT_HEALTH) {
    this.name = name;
    this.health = health;
```
- You should accept health as well, only by default it's 100
- 100 and 50 are magic values, create constants for them